packages/reaction-product-variant/client/templates/products/productDetail/productDetail.js

```diff
@@ -199,17 +199,20 @@ Template.productDetail.events({
       template.$(".title-edit-input").focus();
     }
     const variants = ReactionProduct.getVariants(self._id);
-    for (let variant of variants) {
-      let index = _.indexOf(variants, variant);
+    variants.forEach((variant, index) => {
```

Comment:
Again, why are we wholesale changing all iterators to `forEach`'s? Does this fix a bug?